convert2rhel/actions/system_checks/efi.py

```diff
@@ -78,6 +78,14 @@ def run(self):
                 "The current UEFI bootloader '%s' is not referring to any binary UEFI"
                 " file located on local EFI System Partition (ESP)." % efiboot_info.current_bootnum
             )
+            self.add_message(
+                level="WARNING",
+                id="UEFI_BOOTLOADER_MISMATCH",
+                message=(
+                    "The current UEFI bootloader '%s' is not referring to any binary UEFI"
+                    " file located on local EFI System Partition (ESP)." % efiboot_info.current_bootnum
+                ),
+            )
```

Comment:
Remember to remove the logger.warning above
```suggestion
            message = (
                    "The current UEFI bootloader '%s' is not referring to any binary UEFI"
                    " file located on local EFI System Partition (ESP)." % efiboot_info.current_bootnum
                )
            logger.warning(message)
            self.add_message(
                level="WARNING",
                id="UEFI_BOOTLOADER_MISMATCH",
                message=message,
            )
```